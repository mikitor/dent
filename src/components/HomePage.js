import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const HomePage = props => (
  <div>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default HomePage;
