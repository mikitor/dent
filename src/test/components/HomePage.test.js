import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';

it('should render HomePage component', () => {
  const wrapper = shallow(
    <HomePage
      filters={{ sortBy: 'date', startDate: undefined, endDate: undefined, text: 'test' }}
    />
  );
  expect(wrapper).toMatchSnapshot();
});
