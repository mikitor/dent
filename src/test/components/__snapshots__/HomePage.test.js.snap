// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage component 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
